BaseCollector.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from abc import ABC, abstractmethod
+from os import getenv
+
+import modules.TimedBlacklist as blacklist
+from modules.NetboxHelper import NetboxHelper
+from modules.VCenterConnection import VCenterConnection
+
+logger = logging.getLogger('esxi-exporter')
+
+
+class BaseCollector(ABC):
+
+    def __init__(self) -> None:
+        super().__init__()
+        self.vCenter = VCenterConnection(getenv('VCENTER_URL'), getenv('VCENTER_USER'), getenv('VCENTER_PASSWORD'))

```

Comment:
remove inline camel case everywhere
```suggestion
        self.vcenter = VcenterConnection(getenv('VCENTER_URL'), getenv('VCENTER_USER'), getenv('VCENTER_PASSWORD'))

```

Reply:
what about the classname `VcenterConnection`

Reply:
yes, that's what I've suggested. You can take my suggestion and commit it in github directly